.gitignore

```diff
@@ -11,6 +11,7 @@ test/**/test_output
 
 node_modules/
 npm-debug.log
+.yarn-lock
 
```

Comment:
we should probably ignore `default-packages` as well, so contributors don't accidentally commit theirs.